test/functional/page_objects/time_picker.ts

```diff
@@ -123,8 +123,21 @@ export class TimePickerPageObject extends FtrService {
   /**
    * @param {String} fromTime MMM D, YYYY @ HH:mm:ss.SSS
    * @param {String} toTime MMM D, YYYY @ HH:mm:ss.SSS
+   * @param {Boolean} force time picker force update, default is false
    */
-  public async setAbsoluteRange(fromTime: string, toTime: string) {
+  public async setAbsoluteRange(fromTime: string, toTime: string, force = false) {
```

Comment:
Under what circumstances does it make sense to force?

Reply:
Off the top of my head, anytime flakiness is encountered (timepicker can be notorious sometimes).
This pr is a pr from a former elastic employee (our boss) and we did discuss at some length. 
Further, there is some use of force flag within this pr.

@dmlemeshko may have more to weigh in on this.

Reply:
@andrewctate , it is a good question. For some reason Lee picked area chart, maybe he was doing some testing.
You might know more about this test, and if there is no good reason re-resetting the time picker with already the same range, I can push an update and remove it.

Reply:
@dmlemeshko I agree with Andrew, this test was not flaky. I might miss something here but I would propose to remove it!